### ISSUE TITLE
CI against changes to Autotrain 

### ISSUE THREAD

**Issue by mathemakitten** [huggingface/model-evaluator#54]
We should consider running a tiny, standard evaluation job with this space whenever changes to the Autotrain codebase get merged. The space has gone down few times in the past couple of weeks with bigger changes (deprecated endpoints we use for creating autoevaluate projects) and it'd be nice to get a ping when this happens!

**Comment by lewtun** [comment]
Yes, nice idea! Instead of testing the UI directly (that would require something like `selenium`), a simple alternative would be to have an end-to-end script that launches the job and we poll for success. This could then be triggered with GitHub Actions using the logic you suggest :)

I don't have bandwidth to implement this right now, but I'm more than happy to review a PR if you do!
